File: 艾诗缇/mdx_gyl/androidjson/com/mdx/store/data/proto/GongT.java
//
//  GongT
//
//  Created by liulu on 2014-09-25 18:52:22
//  Copyright (c) liulu All rights reserved.


/**
   
*/

package com.mdx.store.data.proto;

import com.mdx.framework.server.api.json.JsonData;
import org.json.JSONObject;

import java.util.List;
import java.util.ArrayList;


public class GongT {


	/***** 查询库存-返回参数(UserccReturn) *****/
	public static class QueryAllCanBackCPList extends JsonData{
		private static final long serialVersionUID = 1L;

		public List<com.mdx.store.data.proto.GongT.QueryAllCanBackCP > data=new ArrayList<com.mdx.store.data.proto.GongT.QueryAllCanBackCP > (); //评论列表

		public QueryAllCanBackCPList() throws Exception {
		}

		public QueryAllCanBackCPList(JSONObject json) throws Exception {
			this.build(json);
		}

		public void build(JSONObject json) throws Exception{
			getJsonArray(json, "data", com.mdx.store.data.proto.GongT.QueryAllCanBackCP .class, data); //评论列表
		}

	}

	/*****  *****/
	public static class QueryJlJE extends JsonData{
		private static final long serialVersionUID = 1L;

		public String allje; //累计奖励额度
		public String dhje; //兑换额度
		public String ffe; //发放金额
		public String yue; //余额 

		public QueryJlJE() throws Exception {
		}

		public QueryJlJE(JSONObject json) throws Exception {
			this.build(json);
		}

		public void build(JSONObject json) throws Exception{
			allje= getJsonString(json,"allje"); //累计奖励额度
			dhje= getJsonString(json,"dhje"); //兑换额度
			ffe= getJsonString(json,"ffe"); //发放金额
			yue= getJsonString(json,"yue"); //余额 
		}

	}

	/***** 查询库存-返回参数(UserccReturn) *****/
	public static class QueryAllMyOrderList extends JsonData{
		private static final long serialVersionUID = 1L;

		public List<com.mdx.store.data.proto.GongT.QueryAllMyOrder > data=new ArrayList<com.mdx.store.data.proto.GongT.QueryAllMyOrder > (); //评论列表

		public QueryAllMyOrderList() throws Exception {
		}

		public QueryAllMyOrderList(JSONObject json) throws Exception {
			this.build(json);
		}

		public void build(JSONObject json) throws Exception{
			getJsonArray(json, "data", com.mdx.store.data.proto.GongT.QueryAllMyOrder .class, data); //评论列表
		}

	}

	/*****  *****/
	public static class QueryDetailByCode extends JsonData{
		private static final long serialVersionUID = 1L;

		public String cpcode; //商品编码
		public String cpname; //商品名称
		public String cpcount; //商品数量
		public String cpprice; //商品单价
		public String allprice; //商品总额

		public QueryDetailByCode() throws Exception {
		}

		public QueryDetailByCode(JSONObject json) throws Exception {
			this.build(json);
		}

		public void build(JSONObject json) throws Exception{
			cpcode= getJsonString(json,"cpcode"); //商品编码
			cpname= getJsonString(json,"cpname"); //商品名称
			cpcount= getJsonString(json,"cpcount"); //商品数量
			cpprice= getJsonString(json,"cpprice"); //商品单价
			allprice= getJsonString(json,"allprice"); //商品总额
		}

	}

	/***** 查询库存-返回参数(UserccReturn) *****/
	public static class QueryAllMyJLOrderList extends JsonData{
		private static final long serialVersionUID = 1L;

		public List<com.mdx.store.data.proto.GongT.QueryAllMyJLOrder > data=new ArrayList<com.mdx.store.data.proto.GongT.QueryAllMyJLOrder > (); //评论列表

		public QueryAllMyJLOrderList() throws Exception {
		}

		public QueryAllMyJLOrderList(JSONObject json) throws Exception {
			this.build(json);
		}

		public void build(JSONObject json) throws Exception{
			getJsonArray(json, "data", com.mdx.store.data.proto.GongT.QueryAllMyJLOrder .class, data); //评论列表
		}

	}

	/*****  *****/
	public static class QueryJlJECount extends JsonData{
		private static final long serialVersionUID = 1L;

		public String je; //代表总额
		public String kccount; //代表总数量 

		public QueryJlJECount() throws Exception {
		}

		public QueryJlJECount(JSONObject json) throws Exception {
			this.build(json);
		}

		public void build(JSONObject json) throws Exception{
			je= getJsonString(json,"je"); //代表总额
			kccount= getJsonString(json,"kccount"); //代表总数量 
		}

	}

	/*****  *****/
	public static class MakeJFOrder extends JsonData{
		private static final long serialVersionUID = 1L;

		public String retuenmsg; //对应的失败或者成功信息，成功的时候也要提示信息s
		public String returntype; //1 代表成功 2 代表失败
		public String mclass; //商品名称

		public MakeJFOrder() throws Exception {
		}

		public MakeJFOrder(JSONObject json) throws Exception {
			this.build(json);
		}

		public void build(JSONObject json) throws Exception{
			retuenmsg= getJsonString(json,"retuenmsg"); //对应的失败或者成功信息，成功的时候也要提示信息s
			returntype= getJsonString(json,"returntype"); //1 代表成功 2 代表失败
			mclass= getJsonString(json,"class"); //商品名称
		}

	}

	/*****  *****/
	public static class QueryAllMyOrder extends JsonData{
		private static final long serialVersionUID = 1L;

		public String allcount; //总数
		public String allprice; //总额
		public String appdate; //审批日期
		public String appstatus; //发货状态
		public String backdate; //兑换日期
		public String mclass;
		public String connectaddr; //联系地址
		public String connectman; //联系人
		public String connectmoblie; // 联系电话
		public String dbillcode; //单据号
		public String sendcode; //货运单号 
		public String sendcorp; //发货公司
		public String senddate; //发货日期
		public String sendstatus; //发货状态
		public String cpcode; //商品编码
		public String cpname; //商品名称
		public String cpcount; //商品数量
		public String cpprice; //商品单价
		public String allprice; //商品单价
		public String sendcount; //发放金额
		public String surestatus; //确认状态
		public String suredate; //确认日期

		public QueryAllMyOrder() throws Exception {
		}

		public QueryAllMyOrder(JSONObject json) throws Exception {
			this.build(json);
		}

		public void build(JSONObject json) throws Exception{
			allcount= getJsonString(json,"allcount"); //总数
			allprice= getJsonString(json,"allprice"); //总额
			appdate= getJsonString(json,"appdate"); //审批日期
			appstatus= getJsonString(json,"appstatus"); //发货状态
			backdate= getJsonString(json,"backdate"); //兑换日期
			mclass= getJsonString(json,"class");
			connectaddr= getJsonString(json,"connectaddr"); //联系地址
			connectman= getJsonString(json,"connectman"); //联系人
			connectmoblie= getJsonString(json,"connectmoblie"); // 联系电话
			dbillcode= getJsonString(json,"dbillcode"); //单据号
			sendcode= getJsonString(json,"sendcode"); //货运单号 
			sendcorp= getJsonString(json,"sendcorp"); //发货公司
			senddate= getJsonString(json,"senddate"); //发货日期
			sendstatus= getJsonString(json,"sendstatus"); //发货状态
			cpcode= getJsonString(json,"cpcode"); //商品编码
			cpname= getJsonString(json,"cpname"); //商品名称
			cpcount= getJsonString(json,"cpcount"); //商品数量
			cpprice= getJsonString(json,"cpprice"); //商品单价
			allprice= getJsonString(json,"allprice"); //商品单价
			sendcount= getJsonString(json,"sendcount"); //发放金额
			surestatus= getJsonString(json,"surestatus"); //确认状态
			suredate= getJsonString(json,"suredate"); //确认日期
		}

	}

	/*****  *****/
	public static class QueryAllMyJLOrder extends JsonData{
		private static final long serialVersionUID = 1L;

		public String dbillcode; //总数
		public String allcount; //总额
		public String backdate; //审批日期
		public String appstatus; //发货状态
		public String appdate; //兑换日期
		public String sendstatus;
		public String senddate; //联系地址
		public String sendcount; //联系人
		public String surestatus; // 联系电话
		public String suredate; //货运单号 

		public QueryAllMyJLOrder() throws Exception {
		}

		public QueryAllMyJLOrder(JSONObject json) throws Exception {
			this.build(json);
		}

		public void build(JSONObject json) throws Exception{
			dbillcode= getJsonString(json,"dbillcode"); //总数
			allcount= getJsonString(json,"allcount"); //总额
			backdate= getJsonString(json,"backdate"); //审批日期
			appstatus= getJsonString(json,"appstatus"); //发货状态
			appdate= getJsonString(json,"appdate"); //兑换日期
			sendstatus= getJsonString(json,"sendstatus");
			senddate= getJsonString(json,"senddate"); //联系地址
			sendcount= getJsonString(json,"sendcount"); //联系人
			surestatus= getJsonString(json,"surestatus"); // 联系电话
			suredate= getJsonString(json,"suredate"); //货运单号 
		}

	}

	/*****  *****/
	public static class QueryAllCanBackCP extends JsonData{
		private static final long serialVersionUID = 1L;

		public String mclass; //固定不用管
		public String cpcode; //商品编码
		public String cpname; //商品名称
		public String pk_cpkey; //产品Key
		public String cpprice; //产品单价

		public QueryAllCanBackCP() throws Exception {
		}

		public QueryAllCanBackCP(JSONObject json) throws Exception {
			this.build(json);
		}

		public void build(JSONObject json) throws Exception{
			mclass= getJsonString(json,"class"); //固定不用管
			cpcode= getJsonString(json,"cpcode"); //商品编码
			cpname= getJsonString(json,"cpname"); //商品名称
			pk_cpkey= getJsonString(json,"pk_cpkey"); //产品Key
			cpprice= getJsonString(json,"cpprice"); //产品单价
		}

	}

	/***** 查询库存-返回参数(UserccReturn) *****/
	public static class QueryDetailByCodeList extends JsonData{
		private static final long serialVersionUID = 1L;

		public List<com.mdx.store.data.proto.GongT.QueryAllMyOrder > data=new ArrayList<com.mdx.store.data.proto.GongT.QueryAllMyOrder > (); //评论列表

		public QueryDetailByCodeList() throws Exception {
		}

		public QueryDetailByCodeList(JSONObject json) throws Exception {
			this.build(json);
		}

		public void build(JSONObject json) throws Exception{
			getJsonArray(json, "data", com.mdx.store.data.proto.GongT.QueryAllMyOrder .class, data); //评论列表
		}

	}


}

File: 艾诗缇/mdx_gyl/iosJson/GongT.m
//
//  GongT.h
//  GongT
//
//  Created by liulu on 2014-09-25 18:52:22
//  Copyright (c) liulu All rights reserved.
//
//
/**

**/
#import "GongT.h"


/***** 查询库存-返回参数(UserccReturn) *****/
@implementation QueryAllCanBackCPList

@synthesize data; //评论列表

- (void) build:(NSDictionary*)json{
        data=[JsonData getJsonArray:json key:@"data" clas:@"QueryAllCanBackCP" msg:nil]; //评论列表
}


@end

/*****  *****/
@implementation QueryJlJE

@synthesize allje; //累计奖励额度
@synthesize dhje; //兑换额度
@synthesize ffe; //发放金额
@synthesize yue; //余额 

- (void) build:(NSDictionary*)json{
        allje=[JsonData getJsonString:json key:@"allje"]; //累计奖励额度
        dhje=[JsonData getJsonString:json key:@"dhje"]; //兑换额度
        ffe=[JsonData getJsonString:json key:@"ffe"]; //发放金额
        yue=[JsonData getJsonString:json key:@"yue"]; //余额 
}


@end

/***** 查询库存-返回参数(UserccReturn) *****/
@implementation QueryAllMyOrderList

@synthesize data; //评论列表

- (void) build:(NSDictionary*)json{
        data=[JsonData getJsonArray:json key:@"data" clas:@"QueryAllMyOrder" msg:nil]; //评论列表
}


@end

/*****  *****/
@implementation QueryDetailByCode

@synthesize cpcode; //商品编码
@synthesize cpname; //商品名称
@synthesize cpcount; //商品数量
@synthesize cpprice; //商品单价
@synthesize allprice; //商品总额

- (void) build:(NSDictionary*)json{
        cpcode=[JsonData getJsonString:json key:@"cpcode"]; //商品编码
        cpname=[JsonData getJsonString:json key:@"cpname"]; //商品名称
        cpcount=[JsonData getJsonString:json key:@"cpcount"]; //商品数量
        cpprice=[JsonData getJsonString:json key:@"cpprice"]; //商品单价
        allprice=[JsonData getJsonString:json key:@"allprice"]; //商品总额
}


@end

/***** 查询库存-返回参数(UserccReturn) *****/
@implementation QueryAllMyJLOrderList

@synthesize data; //评论列表

- (void) build:(NSDictionary*)json{
        data=[JsonData getJsonArray:json key:@"data" clas:@"QueryAllMyJLOrder" msg:nil]; //评论列表
}


@end

/*****  *****/
@implementation QueryJlJECount

@synthesize je; //代表总额
@synthesize kccount; //代表总数量 

- (void) build:(NSDictionary*)json{
        je=[JsonData getJsonString:json key:@"je"]; //代表总额
        kccount=[JsonData getJsonString:json key:@"kccount"]; //代表总数量 
}


@end

/*****  *****/
@implementation MakeJFOrder

@synthesize retuenmsg; //对应的失败或者成功信息，成功的时候也要提示信息s
@synthesize returntype; //1 代表成功 2 代表失败
@synthesize mclass; //商品名称

- (void) build:(NSDictionary*)json{
        retuenmsg=[JsonData getJsonString:json key:@"retuenmsg"]; //对应的失败或者成功信息，成功的时候也要提示信息s
        returntype=[JsonData getJsonString:json key:@"returntype"]; //1 代表成功 2 代表失败
        mclass=[JsonData getJsonString:json key:@"class"]; //商品名称
}


@end

/*****  *****/
@implementation QueryAllMyOrder

@synthesize allcount; //总数
@synthesize allprice; //总额
@synthesize appdate; //审批日期
@synthesize appstatus; //发货状态
@synthesize backdate; //兑换日期
@synthesize mclass;
@synthesize connectaddr; //联系地址
@synthesize connectman; //联系人
@synthesize connectmoblie; // 联系电话
@synthesize dbillcode; //单据号
@synthesize sendcode; //货运单号 
@synthesize sendcorp; //发货公司
@synthesize senddate; //发货日期
@synthesize sendstatus; //发货状态
@synthesize cpcode; //商品编码
@synthesize cpname; //商品名称
@synthesize cpcount; //商品数量
@synthesize cpprice; //商品单价
@synthesize allprice; //商品单价
@synthesize sendcount; //发放金额
@synthesize surestatus; //确认状态
@synthesize suredate; //确认日期

- (void) build:(NSDictionary*)json{
        allcount=[JsonData getJsonString:json key:@"allcount"]; //总数
        allprice=[JsonData getJsonString:json key:@"allprice"]; //总额
        appdate=[JsonData getJsonString:json key:@"appdate"]; //审批日期
        appstatus=[JsonData getJsonString:json key:@"appstatus"]; //发货状态
        backdate=[JsonData getJsonString:json key:@"backdate"]; //兑换日期
        mclass=[JsonData getJsonString:json key:@"class"];
        connectaddr=[JsonData getJsonString:json key:@"connectaddr"]; //联系地址
        connectman=[JsonData getJsonString:json key:@"connectman"]; //联系人
        connectmoblie=[JsonData getJsonString:json key:@"connectmoblie"]; // 联系电话
        dbillcode=[JsonData getJsonString:json key:@"dbillcode"]; //单据号
        sendcode=[JsonData getJsonString:json key:@"sendcode"]; //货运单号 
        sendcorp=[JsonData getJsonString:json key:@"sendcorp"]; //发货公司
        senddate=[JsonData getJsonString:json key:@"senddate"]; //发货日期
        sendstatus=[JsonData getJsonString:json key:@"sendstatus"]; //发货状态
        cpcode=[JsonData getJsonString:json key:@"cpcode"]; //商品编码
        cpname=[JsonData getJsonString:json key:@"cpname"]; //商品名称
        cpcount=[JsonData getJsonString:json key:@"cpcount"]; //商品数量
        cpprice=[JsonData getJsonString:json key:@"cpprice"]; //商品单价
        allprice=[JsonData getJsonString:json key:@"allprice"]; //商品单价
        sendcount=[JsonData getJsonString:json key:@"sendcount"]; //发放金额
        surestatus=[JsonData getJsonString:json key:@"surestatus"]; //确认状态
        suredate=[JsonData getJsonString:json key:@"suredate"]; //确认日期
}


@end

/*****  *****/
@implementation QueryAllMyJLOrder

@synthesize dbillcode; //总数
@synthesize allcount; //总额
@synthesize backdate; //审批日期
@synthesize appstatus; //发货状态
@synthesize appdate; //兑换日期
@synthesize sendstatus;
@synthesize senddate; //联系地址
@synthesize sendcount; //联系人
@synthesize surestatus; // 联系电话
@synthesize suredate; //货运单号 

- (void) build:(NSDictionary*)json{
        dbillcode=[JsonData getJsonString:json key:@"dbillcode"]; //总数
        allcount=[JsonData getJsonString:json key:@"allcount"]; //总额
        backdate=[JsonData getJsonString:json key:@"backdate"]; //审批日期
        appstatus=[JsonData getJsonString:json key:@"appstatus"]; //发货状态
        appdate=[JsonData getJsonString:json key:@"appdate"]; //兑换日期
        sendstatus=[JsonData getJsonString:json key:@"sendstatus"];
        senddate=[JsonData getJsonString:json key:@"senddate"]; //联系地址
        sendcount=[JsonData getJsonString:json key:@"sendcount"]; //联系人
        surestatus=[JsonData getJsonString:json key:@"surestatus"]; // 联系电话
        suredate=[JsonData getJsonString:json key:@"suredate"]; //货运单号 
}


@end

/*****  *****/
@implementation QueryAllCanBackCP

@synthesize mclass; //固定不用管
@synthesize cpcode; //商品编码
@synthesize cpname; //商品名称
@synthesize pk_cpkey; //产品Key
@synthesize cpprice; //产品单价

- (void) build:(NSDictionary*)json{
        mclass=[JsonData getJsonString:json key:@"class"]; //固定不用管
        cpcode=[JsonData getJsonString:json key:@"cpcode"]; //商品编码
        cpname=[JsonData getJsonString:json key:@"cpname"]; //商品名称
        pk_cpkey=[JsonData getJsonString:json key:@"pk_cpkey"]; //产品Key
        cpprice=[JsonData getJsonString:json key:@"cpprice"]; //产品单价
}


@end

/***** 查询库存-返回参数(UserccReturn) *****/
@implementation QueryDetailByCodeList

@synthesize data; //评论列表

- (void) build:(NSDictionary*)json{
        data=[JsonData getJsonArray:json key:@"data" clas:@"QueryAllMyOrder" msg:nil]; //评论列表
}


@end

